test/integers.test.ts

```diff
@@ -1,10 +1,12 @@
 import * as fc from '../src/index'
+import * as Strategies from '../src/strategies/index'
 import {it} from 'mocha'
 import {expect} from 'chai'
 
 describe('Integer tests', () => {
   it('finds there is a number in the -10, 10 range with inverse and shrink it to 0', () => {
     expect(fc.scenario()
+      .config(new Strategies.RandomCachedStrategy()) // NOTE - This test still passes with a shrinkingless strategy
```

Comment:
Yeah, you are right. I marked the test with that warning for further investigation. However, according to the test itself, I think we should change the test description and also remove the provided `example` since it makes more sense to only use the `example` field when shrinking is part of the test.